src/utils/aquarius.ts

```diff
@@ -107,56 +108,15 @@ export async function getAssetsNames(
   }
 }
 
-export async function getAlgorithmsForAssetSelection(
-  metadataCacheUri: string,
-  selectedAlgorithms?: PublisherTrustedAlgorithm[]
-): Promise<AssetSelectionAsset[]> {
-  const query = {
-    page: 1,
-    query: {
-      query_string: {
-        query: `(service.attributes.main.type:algorithm) -isInPurgatory:true`
-      }
-    },
-    sort: { created: -1 }
-  }
+export async function getAlgorithms(
```

Comment:
don't think we need this abstraction since method does not specifically get algorithms. So wherever `getAlgorithms()` is used, `queryMetadata()` can be used instead

Reply:
good point 👍